src/elements/chatbox/chatbox.js

```diff
@@ -0,0 +1,60 @@
+import './chatbox.less'
+import React, { useState, useEffect } from 'react'
+import url from '/util/url'
+
+export const ChatBox = ({ roomId, ...props }) => {
```

Comment:
I'm not a big fan of using rest for `...props` as it obscures what props the component expects. It looks like you are forwarding them to the textarea, so I'd recommend using a named prop like `ChatBox = ({ roomId, textAreaProps })`